maintainers/maintainer-list.nix

```diff
@@ -4366,6 +4366,12 @@
     githubId = 15692230;
     name = "Muhammad Herdiansyah";
   };
+  koozz = {
+    name = "Jan van den Berg";
+    email = "264371+koozz@users.noreply.github.com";
+    github = "koozz";
+    githubId = 264371;
```

Comment:
I believe the `githubId` is off by one :grimacing: 
```
curl -s https://api.github.com/users/koozz | jq .id
264372
```

Reply:
Weird, I found the noreply-email in my GitHub settings and copied that. Fixed it now.